submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/wiremock/FireMockDatabase.kt

```diff
@@ -28,55 +30,56 @@ class FireMockDatabase(
     }
 
     fun setPath(fireOid: String, path: String) {
-        if (recordsById.values.firstOrNull { it.path == path } != null)
-            throw FireException("Path '$path' has already a file", BAD_REQUEST)
+        val normalizedPath = normalizePath(path)
```

Comment:
According to my understading fire will change the path of 

```
1. /my/path/file.txt
2. my/path/file.txt
3. ///my/path/file.txt
4. /////////my/path/file.txt
```

To `/my/path/file.txt` so if we try to search by any of the previos paths fire will not find file but if we try to set it will. We need to represent the same in the mock. so We can

1.  Save file path and system path when seting the path. Perform all search operation by path and use system path for disk operation.
2. OR Create FireFileSystem that perform operation using path (removing prefix / to make resolve cllas or)

one way or anothe only path operation should be affected as in Fire.